core/trino-spi/src/main/java/io/trino/spi/connector/MergeProcessorUtilities.java

```diff
@@ -20,13 +20,16 @@
 import java.util.Optional;
 import java.util.function.Predicate;
 
-import static io.trino.spi.connector.MergeDetails.DELETE_OPERATION_NUMBER;
-import static io.trino.spi.connector.MergeDetails.INSERT_OPERATION_NUMBER;
 import static io.trino.spi.type.IntegerType.INTEGER;
 import static java.lang.String.format;
 
 public final class MergeProcessorUtilities
 {
+    public static final int DEFAULT_CASE_OPERATION_NUMBER = -1;
```

Comment:
add
```
// Used inside engine only
```

Reply:
If that's the case, please remove from the SPI. Should we filter it out?

Reply:
No need - - it has been removed from the SPI.